composer.json

```diff
@@ -47,7 +47,7 @@
         "ramsey/uuid-doctrine": "^2.0.0",
         "slim/psr7": "^1.3",
         "slim/slim": "^4.7",
-        "stripe/stripe-php": "^9.1.0",
+        "stripe/stripe-php": "^v15.1.0",
```

Comment:
Are the version anchoring semantics the same when `v` is included? I'd probably still avoid it unless there's a reason it's needed as I would probably miss this when e.g. searching a codebase for particular version specs.

Reply:
I think composer ignores the `v` prefix. I hadn't thought about this, happy to remove. Done in af0e4681